database/other/communities.json

```diff
@@ -1 +1,9 @@
-[]
\ No newline at end of file
+[
+    {
+        "name": "Google Developer Groups",
+        "description": "It gives opportunity to learn new skills in a variety of formats. Here multiple events,workshops ,meet-ups also organised virtually or in-person with similar interests in technology",
```

Comment:
[Proofreading]
```suggestion
        "description": "It gives tech professionals opportunity to learn new skills in a variety of formats. Here, events, workshops, meet-ups are organised virtually or in-person for people with similar interests in technology",
```

Reply:
Thanks for feedback, I am changing it definitely.

Reply:
@CBID2 I have updated the description, do check it out  : |)